File: test_resources/dart/consts.dart
const commands = _Commands();
const addr = _Addr();
const groups = _Groups();
const commandsBuffers = _CommandsBuffers();

class _Commands {
  const _Commands();

  final int drawLines = 131073;
  final int drawPath = 131074;
  final int drawQuads = 131075;
  final int drawCenteredQuads = 131076;
  final int drawTexts = 131077;
  final int setColorPipeline = 131078;
  final int setTexturePipeline = 131079;
  final int drawCircles = 131080;
  final int drawHollowCircles = 131081;
}

class _Addr {
  const _Addr();

  final String someValue = "Hello World!";
  final groups = const _Groups();
  final double deltaTime = 16.6;
  final bool flag = true;
  final commandsBuffers = const _CommandsBuffers();
}

class _Groups {
  const _Groups();

  final int main = 0;
  final int mainRender = 1;
  final int rpc = 2;
  final int rpcSync = 3;
  final int rpcRead = 4;
}

class _CommandsBuffers {
  const _CommandsBuffers();

  final int win1MainRender = 0;
}

File: test_resources/kotlin/consts.kt
object Commands {
    const val DRAW_LINES: ULong = 131073UL
    const val DRAW_PATH: ULong = 131074UL
    const val DRAW_QUADS: ULong = 131075UL
    const val DRAW_CENTERED_QUADS: ULong = 131076UL
    const val DRAW_TEXTS: ULong = 131077UL
    const val SET_COLOR_PIPELINE: ULong = 131078UL
    const val SET_TEXTURE_PIPELINE: ULong = 131079UL
    const val DRAW_CIRCLES: ULong = 131080UL
    const val DRAW_HOLLOW_CIRCLES: ULong = 131081UL
}

object Addr {
    const val SOME_VALUE: String = "Hello World!"

    object Groups {
        const val MAIN: ULong = 0UL
        const val MAIN_RENDER: ULong = 1UL
        const val RPC: ULong = 2UL
        const val RPC_SYNC: ULong = 3UL
        const val RPC_READ: ULong = 4UL
    }

    const val DELTA_TIME: Double = 16.6
    const val FLAG: Boolean = true

    object CommandsBuffers {
        const val WIN1_MAIN_RENDER: ULong = 0UL
    }
}
